src/app/pages/payout-page/payout-search/payout-search.component.html

```diff
@@ -0,0 +1,145 @@
+<div class="page-content">
+    <app-content-loader
```

Comment:
We don't have a linter for html (yet), but can you make sure your IDE is configured to use 2 tabs as indentation?